foodsy/Models/Ingredient.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  Ingredient.swift
+//  foodsy
+//
+//  Created by hsherchan on 10/13/17.
+//  Copyright © 2017 Foodly. All rights reserved.
+//
+
+import UIKit
+
+class Ingredient: NSObject {
```

Comment:
@hshercha we can just subclass PFObject and then we will start saving this stuff in Parse: https://guides.codepath.com/ios/Building-Data-driven-Apps-with-Parse#parse-data-objects-pfobject-parse-files-pffile

we can do this as a follow up tho

Reply:
That's fine. I figured we could iterate on this.